flow/project.py

```diff
@@ -569,6 +572,13 @@ def copy_from(cls, *other_funcs):
             @classmethod
             def after(cls, *other_funcs):
                 "True if and only if all post conditions of other operation-function(s) are met."
+                for condition_i in other_funcs:
+                    check_flag = 0
+                    for (name, fun) in cls._parent_class._collect_operations():
```

Comment:
Same thing here as above, not sure about `fun` but a good argument could change my mind.

Reply:
@mikemhenry changed `fun` and `_func`!😅